proto/storage/report_configuration.proto

```diff
@@ -21,7 +21,7 @@ message ReportConfiguration {
     oneof                    filter {
         VulnerabilityReportFilters  vuln_report_filters    = 5;
     }
-    string                    scope_id                     = 6;
+    string                    scope_id                     = 6 [(gogoproto.moretags) = 'search:"Embedded Collection ID"'];
```

Comment:
Can replace the walk on reportConfigDatastore in `DeleteCollection` inside collection service to use search instead.